### DISCUSSION TITLE
Always show items on ground without holding ALT?

### DISCUSSION THREAD

**Discussion by BCullen2201** [diasurgical/DevilutionX#4433] [Q&A] [answered]
I am incredibly lazy and would like to be able to see what items are on the ground around me, without having to use my left hand and hold down the ALT key.

Is there a way to do this? I looked in the `diablo.ini` file, but didn't see anything that would always highlight items on the ground

**Comment by AJenbo** [comment, marked as answer]
Press the right ctrl key

**Reply by BCullen2201** [reply]
Thanks bud!
